### DISCUSSION TITLE
Remote play still not working

### DISCUSSION THREAD

**Discussion by somy05** [redphx/better-xcloud#175] [General]
Remote play was not working for me on version 2.0.1, the creator updated it to 2.0.2 to fix it for free users (I am GPU member) but it still does not work for me. I click remote play, and it shows the xbox logo and plays the sound, then after a few seconds of a black screen it says I was disconnected from the session. I'm using the extension on MacOS edge and my console is definitely working properly, as remote play works from my phone but not the website. Please help

**Comment by redphx** [comment]
> as remote play works from my phone but not the website  

Did it work on your phone with Better xCloud?

**Reply by somy05** [reply]
I’ll try it and let you know, I did it from the Xbox app

**Comment by redphx** [comment]
Moved to #176
